indigo/analysis/refs/provisions.py

```diff
@@ -361,7 +397,8 @@ class ProvisionRefsMatcher(CitationMatcher):
 
     # this just finds the start of a potential match, the grammar looks for the rest
     pattern_names = '|'.join(ProvisionRefsResolver.element_names.keys())
-    pattern_re = re.compile(fr'\b({pattern_names})\s+(\d|\([a-z0-9])', re.IGNORECASE)
+    the_schedule_patterns = '|'.join(['the schedule', 'die bylaag'])
```

Comment:
is the French missing here, or how does this part fit in?

Reply:
we don't currently support french at all when doing lookups - only the grammar supports it. I don't want to change that with this PR.